charts/redpanda/values.yaml

```diff
@@ -77,65 +82,66 @@ console:
     create: false
   config: {}
 
-#
 # Authentication
+# See https://docs.redpanda.com/docs/manage/kubernetes/security/sasl-kubernetes/
 auth:
-  # SASL configuration
   sasl:
-    # When enabling SASL, you are required to have a secret available and referenced in secretRef
+    # If you enable SASL, you must have a Secret referenced in auth.sasl.secretRef.
     enabled: false
-    # if unspecified, mechanism defaults to SCRAM-SHA-512
     mechanism: SCRAM-SHA-512
-    # A secret is expected to exist to contain a file with users in the following format:
-    # <userName>:<password>:<mechanism>
-    # You can create these users in the following way:
-    # create a file with one entry per line in the following format:
-    # <username>:<password>:<mechanism>
+    # The Secret that contains your superuser credentials.
+    # See https://docs.redpanda.com/docs/manage/kubernetes/security/sasl-kubernetes/#use-secrets
     # Ensure there is an empty line at the end of the file.
-    # kubectl -n redpanda create secret generic my-users --from-file=users.txt
-    # Then secretRef is required, regardless if created a users list (below) or not
     secretRef: "redpanda-users"
-    # optional list of users
-    # If not an empty list, these users will be created in a secret whose name is defined in "secretRef"
+    # Optional list of superusers.
+    # These superusers will be created in a secret whose name is defined in auth.sasl.secretRef.
```

Comment:
This needs to say something about the requirement that if this list is empty, the secret that is referenced needs to be pre-created.